### PR TITLE
bumping dependencies

### PR DESCRIPTION
**Pull request by thekp**
**Overall change:** _Bumping 3rd party dependencies_

**Code changes:**

covers the following dependabot  PRs
https://github.com/bbc/psammead/pull/2270, https://github.com/bbc/psammead/pull/2269, https://github.com/bbc/psammead/pull/2268, https://github.com/bbc/psammead/pull/2267, https://github.com/bbc/psammead/pull/2266, https://github.com/bbc/psammead/pull/2265, https://github.com/bbc/psammead/pull/2263, https://github.com/bbc/psammead/pull/2262, https://github.com/bbc/psammead/pull/2261, https://github.com/bbc/psammead/pull/2260, https://github.com/bbc/psammead/pull/2259, https://github.com/bbc/psammead/pull/2258, https://github.com/bbc/psammead/pull/2257, https://github.com/bbc/psammead/pull/2320, https://github.com/bbc/psammead/pull/2321, https://github.com/bbc/psammead/pull/2322, https://github.com/bbc/psammead/pull/2323, https://github.com/bbc/psammead/pull/2324, https://github.com/bbc/psammead/pull/2325, https://github.com/bbc/psammead/pull/2326, https://github.com/bbc/psammead/pull/2327, https://github.com/bbc/psammead/pull/2328, https://github.com/bbc/psammead/pull/2329, https://github.com/bbc/psammead/pull/2330, https://github.com/bbc/psammead/pull/2331, https://github.com/bbc/psammead/pull/2332, https://github.com/bbc/psammead/pull/2333, https://github.com/bbc/psammead/pull/2334, https://github.com/bbc/psammead/pull/2335, https://github.com/bbc/psammead/pull/2336, https://github.com/bbc/psammead/pull/2337, https://github.com/bbc/psammead/pull/2338, https://github.com/bbc/psammead/pull/2339

**Testing:** Regression Testing required.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

**Review by jroebu14 — APPROVED** [review]

**Review by FK78 — APPROVED** [review]
LGTM!

**Review by FK78 — COMMENTED** [review]
Snapshots need updating.

**Review by FK78 — CHANGES_REQUESTED** [review]
There are more dependabot issues.

**Review by FK78 — APPROVED** [review]

File: packages/components/psammead-play-button/CHANGELOG.md [hunk 1]
```diff
@@ -3,4 +3,5 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------------- | ----------- |
+| 1.0.0-alpha.1 | [PR#2318](https://github.com/bbc/psammead/pull/2253) Update snapshot. |
```
Comment (ryanmccombe) [inline packages/components/psammead-play-button/CHANGELOG.md]:
```suggestion
| 1.0.0-alpha.1 | [PR#2318](https://github.com/bbc/psammead/pull/2318) Update snapshot. |
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.8.15 | [PR#2318](https://github.com/bbc/psammead/pull/2298) Bumping 3rd party dependencies |
```
Comment (ryanmccombe) [inline CHANGELOG.md]:
```suggestion
| 1.8.15 | [PR#2318](https://github.com/bbc/psammead/pull/2318) Bumping 3rd party dependencies |
```

**Review by ryanmccombe — APPROVED** [review]
Sick bumps

**Comment by paruchurisilpa** [comment]
Looks good to me..

**Pull request merged**